src/core/app/Server.ts

```diff
@@ -47,7 +47,7 @@ export class Server {
 
   private static matchServerURL(buffer: string | Buffer) {
     const str = buffer.toString().trim();
-    const m = str.match(/Storybook started on => (https?:\/\/.+)/);
+    const m = str.match(/started(?:.|\n)+Local:\s+(https?:\/\/\S+)/);
```

Comment:
Testing this by replacing locally within `node_modules/`. Thank you 🎉 . Looking forward to this being merged and published! Totally blocked with Storybook 4 right now.